index.js

```diff
@@ -0,0 +1,145 @@
+const baseUrl = 'https://trektravel.herokuapp.com/trips'
+
+// Get the modal
+const modal = document.getElementById('myModal');
+
+// Get the button that opens the modal
+const btn = document.getElementById("myBtn");
```

Comment:
You should be using jQuery selectors for these. Also, any code that relies on the DOM should be placed inside of `$(document).ready`, or some function called from there.